README.md

```diff
@@ -7,17 +7,31 @@ Black belt is collection of scripts, tools and guidelines used for developing pr
 
 Please refer to [The Black Belt Documentation](http://black-belt.readthedocs.org/).
 
+```
```

Comment:
I think this should be moved down to the develop black-belt section. It is not required for installation and usage?

Reply:
Good point.